doc/README.txt

```diff
@@ -3,7 +3,7 @@ sphinx.
 
 The necessary python packages have to be installed, e.g. with pip:
 
-    pip install -r requirements.txt
+    pip install nox
```

Comment:
This README refers to building the docs. For that, installing that directory's `requirements.txt` is in fact required. However, we could add a note that most of the time calling the top-level `nox -s doc` will be sufficient.

Reply:
Te requirements.txt was restored so I'll revert the change and add the nox -s doc.